clar2wasm/src/words/consensus_buff.rs

```diff
@@ -125,6 +125,8 @@ impl ComplexWord for FromConsensusBuff {
         // Traverse the input buffer, leaving the offset and length on the stack.
         generator.traverse_expr(builder, args.get_expr(1)?)?;
 
+        // TODO: true, true is too big; related issue: #362
```

Comment:
Actually, it's not even the same as issue 362. Here, we should have a dedicated function to compute exactly how much extra space is needed. For example, deserializing a buffer doesn't require any space since we reuse the bytes from the serialization.

Reply:
Alright, should we open an issue for this and remove the comment?

Reply:
Yes, that will be an optimization issue. Just replace the related issue number with the new one you will create.